docs/api/carts.rst

```diff
@@ -0,0 +1,764 @@
+| id                | Id of cart item unit                                                 |
++-------------------+----------------------------------------------------------------------+
+| adjustments       | List of adjustments related to unit                                  |
++-------------------+----------------------------------------------------------------------+
+| adjustments_total | Sum of all units adjustments                                         |
++-------------------+----------------------------------------------------------------------+
+
+And each adjustment will be build as follows:
+
++--------+--------------------------------------------------------+
+| Field  | Description                                            |
++========+========================================================+
+| id     | Id of cart item unit                                   |
++--------+--------------------------------------------------------+
+| type   | Type of an adjustment (E.g. *order_promotion* or *tax* |
++--------+--------------------------------------------------------+
+| label  | Label of adjustment                                    |
++--------+--------------------------------------------------------+
+| amount | Amount of adjustment                                   |
++--------+--------------------------------------------------------+
+
+.. _Customer detailed serialization: http://docs.sylius.org/en/latest/api/customers.html#customers-api
+.. _Default channel serialization: http://docs.sylius.org/en/latest/api/channels.html#channels-api
+
+.. note::
+
+    If it is still confusing to you, you should read more about `Carts (Orders)`_ and `Adjustments`_ first.
+
+.. _Carts (Orders): http://docs.sylius.org/en/latest/components/Order/models.html#order
+.. _Adjustments: http://docs.sylius.org/en/latest/book/orders/adjustments.html
+
+Creating a Cart
+---------------
+
+Definition
+..........
+
+.. code-block:: text
+
+    POST /api/v1/carts/
+
++---------------+----------------+----------------------------------------------------+
+| Parameter     | Parameter type | Description                                        |
++===============+================+====================================================+
+| Authorization | header         | Token received during authentication               |
++---------------+----------------+----------------------------------------------------+
+| customer      | request        | Email of related customer                          |
++---------------+----------------+----------------------------------------------------+
+| channel       | request        | Code of related channel                            |
++---------------+----------------+----------------------------------------------------+
+| locale_code   | request        | Code of locale in which the cart should be created |
++---------------+----------------+----------------------------------------------------+
+
+Example
+.......
+
```

Comment:
A description of what it does is needed.

```
To create a new cart for the ``shop@example.com`` user in the ``US_WEB`` channel in the ``en_US`` locale use the below method. 

This method replicates the environment of shop usage, when a user is logged in some channel with some locale chosen.
```

I think such descriptions are required somewhere. The API calls are not self-understandable to everyone probably.